docs/features/asc1/teal/index.md

```diff
@@ -13,17 +13,17 @@ Some of the opcodes in TEAL are only valid for a specific contract type. These a
     When writing smart contracts, make sure to follow [TEAL guidelines](../../../reference/teal/guidelines.md). This is very important in order to prevent smart contracts from being compromised.
 
 # TEAL Versions
-Currently Algorand supports version 1 and version 2 of TEAL. When writing TEAL Version 2 contracts make sure to add `#pragma version 2` as the first line of the program. If this line does not exists, the protocol will treat the contract as a version 1 contract. If upgrading a contract to version 2 it is important to verify you are checking the `RekeyTo` property of all transactions that are attached to the contract. See the [TEAL guidelines](../../../reference/teal/guidelines.md) for more suggestions to help prevent a contract from being compromised. 
+Currently, Algorand supports versions 1 - 4 of TEAL. When writing TEAL Version 2 or higher contracts make sure to add `#pragma version #` where # should be replaced by the specific number, as the first line of the program. If this line does not exists, the protocol will treat the contract as a version 1 contract. If upgrading a contract to version 2 or higher it is important to verify you are checking the `RekeyTo` property of all transactions that are attached to the contract. See the [TEAL guidelines](../../../reference/teal/guidelines.md) for more suggestions to help prevent a contract from being compromised. 
```

Comment:
A few suggestions:
"versions 1 - 4" --> "versions 1 through 4"
"when writing TEAL Version 2 or higher contracts make sure to..." --> "When writing contracts with TEAL version 2 or higher, make sure to..."

"If this line does not exists" --> "If this line does not exist"

"If upgrading a contract to version 2 or higher it" --> "If upgrading a contract to version 2 or higher, it"